package.json

```diff
@@ -47,22 +47,22 @@
     "@enact/spotlight": "^4.5.2",
     "@enact/ui": "^4.5.2",
     "@enact/webos": "^4.5.2",
-    "classnames": "^2.3.1",
+    "classnames": "^2.3.2",
     "invariant": "^2.2.4",
     "prop-types": "^15.8.1",
     "ramda": "^0.28.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "warning": "^4.0.3",
-    "ilib": "14.14.0"
+    "ilib": "^14.15.1"
   },
   "peerDependencies": {
-    "ilib": "14.14.0 || ^14.14.0-webos1"
+    "ilib": "^14.15.1 || ^14.15.1-webos1"
   },
   "devDependencies": {
-    "@enact/docs-utils": "0.3.6",
-    "@enact/ui-test-utils": "^1.0.0",
+    "@enact/docs-utils": "0.4.0",
```

Comment:
Any reason to omit the ^?

Reply:
No, this is an omission. Thank you for the check.